packages/patternlab/styleguide/source/_patterns/02-molecules/emergency-header.twig

```diff
@@ -1,8 +1,12 @@
 {% set alertPrefix = emergencyHeader.prefix ?: 'Emergency Alerts' %}
-<h2 class="ma__emergency-header">
+<div class="ma__emergency-header ma__emergency-header--pt">

```

Comment:
what does `pt` stand for? and why do we need to add a class?

Reply:
PatternLab